gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/logical_model/ldm.py

```diff
@@ -42,6 +42,41 @@ def modify_mapped_data_source(self, data_source_mapping: dict) -> None:
                     if data_source_id in data_source_mapping:
                         dataset.data_source_table_id.data_source_id = data_source_mapping[data_source_id]
 
+    @staticmethod
+    def _change_case(object_name: str, lower_case: bool, upper_case: bool) -> str:
+        if lower_case:
+            return object_name.lower()
+        elif upper_case:
+            return object_name.upper()
+        else:
```

Comment:
What about proposing only one optional flag - lowercase or uppercase?
e.g. 
lower_case=True -> object_name.lower()
lower_case=False -> object_name.upper()
lower_case=None (default) –> do nothing

Reply:
Fixed.
I decided to not support the "None"(noop) option, it makes no sense.
The default is upper_case=True because migration to Snowflake is more probable than the opposite.